tests/plugins/os/unix/log/test_messages.py

```diff
@@ -105,3 +106,21 @@ def test_unix_log_messages_malformed_log_year_rollover(target_unix_users, fs_uni
 
         results = list(target_unix_users.messages())
         assert len(results) == 2
+
+
+def test_unix_messages_cloud_init(target_unix_users: Target, fs_unix: VirtualFilesystem) -> None:
+    messages = """
+    2005-08-09 11:55:21,000 - foo.py[DEBUG]: This is a cloud-init message!
+    2005-08-09 11:55:21,001 - util.py[DEBUG]: Cloud-init v. 1.2.3-4ubuntu5 running 'init-local' at Tue, 9 Aug 2005 11:55:21 +0000. Up 13.37 seconds.
+    """
```

Comment:
This line is to long, so flake8 will complain
```suggestion
    """  # noqa: E501
```